helpers/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  expectThrow: async promise => {
+    try {
+      await promise
+    } catch (error) {
+      // console.warn(error.message)
+      // TODO: Check jump destination to destinguish between a throw
+      //       and an actual invalid jump.
+      const invalidJump = error.message.search('invalid JUMP') >= 0
+
+      // TODO: Found out in which circumstances 'invalid opcode' is throwed
+      const invalidOpcode = error.message.search('invalid opcode') >= 0
+
+      // TODO: When we contract A calls contract B, and B throws, instead
```

Comment:
Good observation. This article answers your question: http://truffleframework.com/tutorials/testing-for-throws-in-solidity-tests

Please remove this TODO, if you like, you could add an issue to differentiate between an OOG and failed message call.